src/Api/Request.purs

```diff
@@ -226,37 +226,37 @@ type AuthFieldsRep box r = ( email :: Email, password :: box String | r )
 -- | type RegisterFields = { email :: Email, password :: String, username :: Username }
 -- | type LoginFields = { email :: Email, password :: String }
 -- | ```
--- | 
+-- |
 -- | Admittedly, it's not strictly necessary to share fields among the three auth types because
 -- | they're so small. But this pattern is common in production applications and in libraries like
 -- | `purescript-record`, `purescript-variant`, and `purescript-halogen-formless`, so it's
 -- | important to be aware of it.
 type RegisterFields = { | AuthFieldsRep Unlifted (username :: Username) }
 type LoginFields = { | AuthFieldsRep Unlifted () }
 
--- | This function logs a user in (if they exist), returning an auth token and the user's 
+-- | This function logs a user in (if they exist), returning an auth token and the user's
 -- | minimal profile.
 login :: forall m. MonadAff m => BaseURL -> LoginFields -> m (Either String (Tuple Token Profile))
-login baseUrl fields = 
-  let method = Post $ Just $ encodeJson { user: fields } 
+login baseUrl fields =
+  let method = Post $ Just $ encodeJson { user: fields }
    in requestUser baseUrl { endpoint: Login, method }
 
 -- | This function registers a user (if they don't already exist), returning an auth token and the
 -- | user's minimal profile.
 register :: forall m. MonadAff m => BaseURL -> RegisterFields -> m (Either String (Tuple Token Profile))
-register baseUrl fields = 
+register baseUrl fields =
   let method = Post $ Just $ encodeJson { user: fields }
-   in requestUser baseUrl { endpoint: Users, method } 
+   in requestUser baseUrl { endpoint: Users, method }
 
--- | The login and registration requests share the same underlying implementation, just a different 
+-- | The login and registration requests share the same underlying implementation, just a different
 -- | endpoint. This function can be re-used by both requests.
 requestUser :: forall m. MonadAff m => BaseURL -> RequestOptions -> m (Either String (Tuple Token Profile))
 requestUser baseUrl opts = do
   res <- liftAff $ request $ defaultRequest baseUrl Nothing opts
-  pure $ decodeAuthProfile =<< decodeAt "user" =<< lmap printResponseFormatError res.body
+  pure $ decodeAuthProfile =<< decodeAt "user" =<< bimap printError _.body res
```

Comment:
This line and the two changes in imports are the only actual code changes in this file. The rest is just noise from my editor removing trailing spaces.